CHANGELOG.md

```diff
@@ -203,4 +204,4 @@
 
 Preview release of **GLIDE for Redis** a Polyglot Redis client.
 
-See the [README](README.md) for additional information.
+See the [README](README.md) for additional information.
```

Comment:
I really tried to found what's the diff here... 🥴

Reply:
Its just got one line down, i also don't know why it showing it like its a diff

Reply:
Like i added a line, so it got down by line. Like dhaa